docs/how-to-guides/format-api.md

```diff
@@ -125,13 +125,13 @@ registerFormatType( 'my-custom-format/sample-output', {
 
 Confirm it is working: first build and reload, then make a text selection and click the button. Your browser will likely display that selection differently than the surrounding text.
 
-You can also confirm by switching to HTML view (Code editor Ctrl+Shift+Alt+M) and see the text selection wrapped with `<samp>` HTML tags.
+You can also confirm by switching to HTML view (Code editor `Ctrl+Shift+Alt+M`) and see the text selection wrapped with `<samp>` HTML tags.
 
 Use the `className` option when registering to add your own custom class to the tag. You can use that class and custom CSS to target that element and style as you wish.
 
 ### Step 4: Show the button only for specific blocks (Optional)
 
-By default, the button is rendered on every rich text toolbar (image captions, buttons, paragraphs, etc). You can render the button only on blocks of a certain type by using `wp.data.withSelect` together with `wp.compose.ifCondition`.
+By default, the button is rendered on every rich text toolbar (image captions, buttons, paragraphs, etc). You can render the button only on blocks of a certain type by using `wp.data.select`.
```

Comment:
I think this may be slightly confusing. It looks like it is telling you to use a specific api here because it references code but in fact it is just telling you to use the data API. 

I would suggest we change this line to say "using the data api" 

Ideally even linking to the data api reference.

Reply:
Fixed by 4d729c0
Is this the appropriate link?

Reply:
That is a great question. I just looked and am surprised that there isn't a great landing page for the Data API in general. 

Maybe this here makes more sense: https://developer.wordpress.org/block-editor/reference-guides/data/data-core-block-editor/ ? 

I'm not sure

Reply:
That page mentions `core/block-editor`, but my understanding is that the data API refers to `@wordpress/data`, not `@wordpress/block-editor`.
Unfortunately, the current handbook does not appear to have a page on the data package 🤔

Reply:
Yeah you are right. In that case I would link to here: https://developer.wordpress.org/block-editor/reference-guides/packages/packages-data (just without the anchor at theend 
)

Reply:
Thank you! Indeed, the anchor may not be necessary.
I have updated it and would like to merge it once it passes the test 🙂